firestore-send-email/extension.yaml

```diff
@@ -104,11 +104,12 @@ params:
   - param: DEFAULT_FROM
     type: string
     label: Default FROM address
-    validationRegex: ^\S+@\S+\.\S+$
+    validationRegex: ^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$|^.*<(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})>$
     validationErrorMessage: Must be a valid email address
     required: true
     description: >-
       The email address to use as the sender's address (if it's not specified in the added email document).
+    example: Friendly Firebaser <foobar@example.com>  
```

Comment:
Is the main use case that a dev will want to use a name + email address here?  If not, I would leave the example with just the email address so that a dev doesn't think that they need to use the name, too.

_Suggested change:_

description: >-
  The email address to use as the sender's address (if it's not specified in the added email document). You can optionally include a name with the email address (`Friendly Firebaser <foobar@example.com>`).

example: foobar@example.com